eng/Versions.props

```diff
@@ -82,5 +82,19 @@
     Versions below this comment are not managed by automation and can be changed as needed.
   -->
   <PropertyGroup Label="Manual">
+    <!-- Compatibility with VS 17.4/.NET SDK 7.0.1xx -->
```

Comment:
@RussKie Is there a more elegant solution here?

Reply:
It doesn't look we're doing much different from dotnet/runtime, so I guess this is as elegant as it gets... There's no direct way of knowing what currently supported versions of VS are targeting.